### ISSUE TITLE
chemical identifiers in biolink

### ISSUE THREAD

**Issue by cmungall** [biolink/biolink-model#340]
Currently we have

```
    id_prefixes:
      - CHEBI
      - CHEMBL.COMPOUND
      - DRUGBANK
      - PUBCHEM
      - MESH
      - HMDB
      - INCHI
      - INCHIKEY
      - UNII
      - KEGG
      - GTOPDB
```

but this prioritization has not been vetted or approved. Why is inchi better than inchikey (the latter seems more widely used)? no SMILES? why is mesh preferred over inchi?

Of course, the choice will always be dictated by use case. Sometimes we care about conflation/lumping over protonation state, sometimes we want to split. In theory chebi can provide a representation at the appropriate level of granularity, but it is less complete. However, we can make requests of chemicals for them to prioritize.

I think it may be worthwhile having some narrative documentation about representation here, with some of the basic info: e.g. is a SMILES representation guaranteed to be the same or does it vary with program?

**Comment by nlharris** [comment]
Is this done?

**Comment by nlharris** [comment]
Vlado says this was addressed in Biolink 2.0 and can be closed.
